project-base/app/schema.graphql

```diff
@@ -935,11 +935,11 @@ type LoginResult {
 
 "One of the possible methods of the customer user login"
 enum LoginTypeEnum {
-  admin
+  web
   facebook
   google
   seznam
-  web
+  admin
```

Comment:
Do you have all composer dependencies updated? This seems weird

Reply:
yep, I ran `composer update` and then generated schema and nothing changed 🤷🤷‍♂️🤷‍♀️ Should I do something different?

Reply:
It is reverted when i run `make generate-schema` on my machine. Try to update all dependencies. Contact me if your problem persists.